src/main.rs

```diff
@@ -28,7 +28,7 @@ fn set_realtime() {
 fn find_config() -> Result<String, Error> {
     let config_file_name = "buzzd.json";
     let home_direcory = env::var("HOME");
-    let global_config = format!("/etc/{}", config_file_name);
```

Comment:
I *did* run clippy and fmt, but neither complained about this. Was this a tool finding or did your Rustacean instincts trigger?

Reply:
OK, let's see what happens with the next CI run then 🤔  I did run fmt/clippy on ubuntu-22.04 (as does the workflow you added).

Reply:
There is a green ✅ next to my commit. Seems it was happy.